pkg/process/config/config.go

```diff
@@ -262,37 +262,22 @@ func getContainerHostType() model.ContainerHostType {
 	return model.ContainerHostType_notSpecified
 }
 
+// loadEnvVariable reads env variables specific to process-agent and overrides the corresponding settings
+// in the global Config object.
+// This function is used to handle historic process-agent env vars. New settings should be
+// handled directly in the /pkg/config/process.go file
 func loadEnvVariables() {
 	// The following environment variables will be loaded in the order listed, meaning variables
 	// further down the list may override prior variables.
 	for _, variable := range []struct{ env, cfg string }{
-		{"DD_PROCESS_AGENT_CONTAINER_SOURCE", "process_config.container_source"},
-		{"DD_SCRUB_ARGS", "process_config.scrub_args"},
-		{"DD_STRIP_PROCESS_ARGS", "process_config.strip_proc_arguments"},
 		{"DD_ORCHESTRATOR_URL", "orchestrator_explorer.orchestrator_dd_url"},
```

Comment:
You might be able to move this to [here](https://github.com/DataDog/datadog-agent/blob/f0a67887696ed50b0be9f78d7a5f16fee3a7f0aa/pkg/config/config.go#L908), as long as you get permission from containers

Reply:
To be honest, I'd prefer to move orchestrator related settings in a follow up PR and keep this one focused on processes ones. What do you think?

Reply:
I think that's a valid approach! Let's do that